components/partials/OverviewInfo.tsx

```diff
@@ -33,13 +33,13 @@ export const OverviewInfo = (chapter: Chapter) => {
         onChange={setActiveTab}
         classes="mt-8"
       />
-      <div className="flex grow">
-        {activeTab == 'info' &&
+      <div className="flex">
+        <div className={'tab-content ' + (activeTab != 'info' ? '' : 'tab-content-active')}>
```

Comment:
Could you please also add `aria-hidden` attributes? Right now, the invisible tab content is still being picked up by screen readers. With `aria-hidden=true`, it will be ignored. [More info here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden).

Reply:
I have created a ternary statement based on screen width (1023px), for greater than 1023px it will use the invisible logic and for less than 1023px it will use the existing tab-switching logic.